### ISSUE TITLE
FLAC

### ISSUE THREAD

**Issue by ericdanielski** [dkim0419/SoundRecorder#3]
It is possible to integrate FLAC?

**Comment by GreenLunar** [comment]
Or OGA (OGG), OPUS and SPEEX too.

**Comment by dkim0419** [comment]
I will definitely try and look into that, thanks for the suggestions!

**Comment by KOLANICH** [comment]
+1
Ps, speex is pbsolete, i think some codec designed for 3g or voip vould be better

**Comment by GreenLunar** [comment]
> speex is pbsolete

It is the best one for voice archiving and transmission. You get more fo very much less. Even WMA can't get so much voice information the same as SPEEX does.

> i think some codec designed for 3g or voip vould be better

I assume you refer to CELT or OPUS.

**Comment by GreenLunar** [comment]
Also, SPEEX comes from independent and special-interest-free people. whereas OPUS comes from a giants such as Mozilla and its NSA-friendly sugar-daddy.

**Comment by ericdanielski** [comment]
Silk from Skype and Speex from Xiph are parts of Opus. Opus is also from Xiph.

http://www.xiph.org/press/2013/Opus_1.1/

Opus is better than Ogg Vorbis, so there is no need for Ogg Vorbis. Flac is good as lossless codec.

**Comment by GreenLunar** [comment]
> Opus is also from Xiph.

Yes, but the timing, around big WebRTC announcement, is odd.  I ask myself why then, and not before, while it was SPEEX that has been used by so many VoIP clients such as Ekiga, Jitsi, Linphone, Psi and many more for many years.

**Comment by eadmaster** [comment]
I vote for FLAC support too!

Opus is meant as a Speex (and other lossy codecs) replacement, so it would be nice to have as alternative to AAC.

**Comment by RoestVrijStaal** [comment]
Which audio codec is used now any way? MPC:HC, VLC and Audacity can't open it. Very likely it's a non-FOSS codec.

**Comment by fabianrost84** [comment]
I think it is MPEG AAC Audio at the moment.
